packages/storefront/packages/next/CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Change log
 
+## 1.1.0
+
+- **[ADDED]** Add `middlewareModule` to `createSdk` params.
```

Comment:
I think based on Filip's RFC, we could have

```
**[ADDED]** `middlewareModule` to `createSdk` params.
```

instead, so you read the whole line (including the tag) as a sentence

https://github.com/vuestorefront/vue-storefront/blob/main/packages/changesets/reference-changelog.md#minor-changes